modules/osd-persistent-storage-csi-efs-sts.adoc

```diff
@@ -21,11 +21,11 @@ If you perform this procedure after installing the driver and creating volumes,
 
 * You have access to the cluster as a user with the cluster-admin role.
 * AWS account credentials
-* You have installed the AWS EFS CSI Operator.
+* You have installed the Amazon EFS CSI Operator.
```

Comment:
Revoked.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code. "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html